pfcpiface/p4rt_translator.go

```diff
@@ -19,21 +19,23 @@ import (
 const (
 	DirectionUplink = 1
 
-	FieldAppIPProto       = "app_ip_proto"
-	FieldAppL4Port        = "app_l4_port"
-	FieldAppIPAddress     = "app_ip_addr"
-	FieldIPv4DstPrefix    = "ipv4_dst_prefix"
-	FieldN3Address        = "n3_address"
-	FieldUEAddress        = "ue_address"
-	FieldApplicationID    = "app_id"
-	FieldTEID             = "teid"
-	FieldQFI              = "qfi"
-	FieldCounterIndex     = "ctr_idx"
-	FieldTrafficClass     = "tc"
-	FieldTunnelPeerID     = "tunnel_peer_id"
-	FieldTunnelSrcAddress = "src_addr"
-	FieldTunnelDstAddress = "dst_addr"
-	FieldTunnelSrcPort    = "sport"
+	FieldAppIPProto        = "app_ip_proto"
+	FieldAppL4Port         = "app_l4_port"
+	FieldAppIPAddress      = "app_ip_addr"
+	FieldIPv4DstPrefix     = "ipv4_dst_prefix"
+	FieldN3Address         = "n3_address"
+	FieldUEAddress         = "ue_address"
+	FieldApplicationID     = "app_id"
+	FieldTEID              = "teid"
+	FieldQFI               = "qfi"
+	FieldCounterIndex      = "ctr_idx"
+	FieldTrafficClass      = "tc"
+	FieldTunnelPeerID      = "tunnel_peer_id"
+	FieldTunnelSrcAddress  = "src_addr"
+	FieldTunnelDstAddress  = "dst_addr"
+	FieldTunnelSrcPort     = "sport"
+	FieldSessionMeterIndex = "session_meter_id"
+	FieldAppMeterIndex     = "app_meter_id"
```

Comment:
```suggestion
	FieldSessionMeterIndex = "session_meter_idx"
	FieldAppMeterIndex     = "app_meter_idx"
```
Requires to update the p4info and bmv2.json as well. See: https://github.com/omec-project/up4/pull/219